### DISCUSSION TITLE
Sorting feature for the reading history page

### DISCUSSION THREAD

**Discussion by RajvirSingh1313** [dailydotdev/daily#370] [Feature Requests]
## Feature Request
Sorting Feature for the [reading history page](https://app.daily.dev/history)

### Description
When the first time, I got to know about the history feature, I just wanted to see my first read with dev.to . But I had to scroll all the way down to find it, and it was pretty hard as I read a lot of articles every day. So this is the problem.

### Solution
The solution can be just to add a sorting feature, more like the sorting feature on the main page which helps you to see articles of last week, last month, or last year. With just a little change for the reading history page, like oldest, newest, last week, last month, last year.

I think it would just be good to have it there, I know it isn't that important.

Have a good day

**Comment by idoshamun** [comment]
Thanks a lot for the suggestion @RajvirSingh1313!
We will keep you posted :)

**Reply by RajvirSingh1313** [reply]
Sure, Thanks to whole team of dev.to to make this awesome extension

**Reply by idoshamun** [reply]
You bet! But we're the daily.dev team 🙈😅

**Reply by RajvirSingh1313** [reply]
I already know that 😃
